OPUNetTransportLayer.cpp

```diff
@@ -1128,9 +1122,8 @@ bool OPUNetTransportLayer::DoImmediateProcessing(Packet &packet, sockaddr_in &fr
 			{
 				tlMessage.tlHeader.commandType = tlcJoinGranted;
 				// **DEBUG**
-				logFile << "Client join accepted: ";
-				LogAddress(from);
-				logFile << " (" << tlMessage.joinReply.newPlayerNetID << ")" << std::endl;
+				Log("Client join accepted: " + FormatAddress(from) + 
+					" (" + std::to_string(tlMessage.joinReply.newPlayerNetID) + ")");

```

Comment:
Could potentially change this to use the `FormatPlayerNetID`. I understand why you'd want to do that separately though, as that would change the output slightly.